src/omv/modules/py_sensor.c

```diff
@@ -705,6 +705,24 @@ static mp_obj_t py_sensor_ioctl(uint n_args, const mp_obj_t *args) {
         }
         #endif
 
+        #if (OMV_ENABLE_OV7725 == 1)
```

Comment:
There's no need to gate this by sensor, if an ioctl is not supported it will raise an exception, when it's implemented for another sensor we don't need to remember removing this line (same for the QSTR below).

Reply:
I see you removed it in the following commit.